src/tests/integration/webSocket/webSocket.spec.ts

```diff
@@ -10,10 +10,13 @@ describe('WebSocket Action', () => {
         'src/tests/integration/webSocket/resources/environment';
     const environment = 'config';
 
-    const wss = new WebSocket.Server({ port: 8080, path: '/ws' });
-    wss.on('connection', ws => {
-        ws.on('message', message => {
-            ws.send(message);
+    let wss;
```

Comment:
Please add a type annotation (I suppose it needs to be `let wss: WebSocket.Server | undefined`, which also means that we need to do `wss && wss.close();` in `after`.